### ISSUE TITLE
Unable to get the integration to start. 

### ISSUE THREAD

**Issue by sheldondickinson** [briis/unifiprotect#270]
Randomly, I can't get this integration to start and the integration shows 'failed to set up' message, with the below in the logs.

I have tried rolling back to a v0.8.9, and both v0.9.0-beta1 and v0.9.0-beta2 without luck. 

```
This error originated from a custom integration.

Logger: homeassistant.config_entries
Source: custom_components/unifiprotect/data.py:48 
Integration: Unifi Protect (documentation, issues) 
First occurred: 3:30:26 PM (1 occurrences) 
Last logged: 3:30:26 PM

Error setting up entry UniFi-CloudKey-Gen2-Plus for unifiprotect
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 293, in async_setup
    result = await component.async_setup_entry(hass, self)  # type: ignore
  File "/config/custom_components/unifiprotect/__init__.py", line 99, in async_setup_entry
    await protect_data.async_setup()
  File "/config/custom_components/unifiprotect/data.py", line 32, in async_setup
    await self.async_refresh()
  File "/config/custom_components/unifiprotect/data.py", line 48, in async_refresh
    await self._protectserver.update(
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/unifi_protect_server.py", line 123, in update
    await self._get_device_list(not self.ws_connection)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/unifi_protect_server.py", line 314, in _get_device_list
    self._process_cameras_json(json_response, server_id, include_events)
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/unifi_protect_server.py", line 335, in _process_cameras_json
    process_camera(
  File "/usr/local/lib/python3.9/site-packages/pyunifiprotect/unifi_data.py", line 349, in process_camera
    "image_width": image_width,
UnboundLocalError: local variable 'image_width' referenced before assignment
```

Hopefully this is just me being stupid!

**Comment by briis** [comment]
Thanks. I have seen this error a few times, and thought i fixed it. But I will give it another go.

**Comment by briis** [comment]
Should be fixed when releasing 0.9.0 - soon.
